controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2021,28 +2032,40 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
             std::cout << __func__ << "ERROR: results_count(" << results_count << ")"
                       << " > max_results_size(" << max_results_size << ")" << std::endl;
         } else if ((status == 0) && (results_count > 0)) {
-            //todo: replace the temporary debug print with actual prints of the new results structure
             for (size_t i = 0; i < results_count; i++) {
                 auto res = results[i];
                 std::cout << "result[" << i << "]:" << std::endl
```

Comment:
Instead of this whole big print, why not add `inline std::ostream &operator<<(std::ostream &os, const BML_NEIGHBOR_AP &neigh);` to `struct BML_NEIGHBOR_AP` in `bml_defs.h` then you can replace this with:
```
for (size_t i = 0; i < results_count; i++) {
    std::cout << "result[" << i << "]:" << std::endl << results[i] << std::endl;
}
```